Autopark/Autopark.Presentation/appsettings.json

```diff
@@ -1,6 +1,7 @@
 {
   "ConnectionStrings": {
-    "Default" : "Server=.\\SQLEXPRESS;Initial Catalog=cargot_autopark;Trusted_Connection=True;TrustServerCertificate=True"
+    "Default" : "Server=mssql,1433;Initial Catalog=cargot_autopark;User ID=${DB_USERNAME};Password=${DB_PASSWORD};TrustServerCertificate=True",
+    "Local": "Server=.\\SQLEXPRESS;Initial Catalog=cargot_autopark;Trusted_Connection=True;TrustServerCertificate=True"
```

Comment:
you need to create separate appsettings files. One for production (for docker) and another one for development and store in these files all necessary configurations. This way you you dont need the GetConfiguredConnectionString() as it will be automatically selected the necessary appsettings file with with the correct connection strings. You just need to set the environment in docker compose.